kernel/packages/unity-interface/delightedSurvey.ts

```diff
@@ -32,10 +32,10 @@ function delightedSurvey() {
       const profile = getCurrentUserProfile(globalThis.globalStore.getState())
       if (profile) {
         const payload = {
-          email: profile.ethAddress + '@dcl.gg',
+          email: profile.userId + '@dcl.gg',
           name: profile.name || 'Guest',
           properties: {
-            ethAddress: profile.ethAddress,
+            ethAddress: profile.userId,
```

Comment:
why is ethAddres also userId?

Reply:
I updated the property we send to be called now `userId`, and validated with @2fd that that property name can be changed without risk =), fixing confusions with the previous implementation as you pointed out.